src/main/fc/runtime_config.h

```diff
@@ -85,11 +85,11 @@ typedef enum {
 //    GPS_HOME_MODE   = (1 << 4),
 //    GPS_HOLD_MODE   = (1 << 5),
     HEADFREE_MODE   = (1 << 6),
-//    UNUSED_MODE     = (1 << 7), // old autotune
-    PASSTHRU_MODE   = (1 << 8),
-//    RANGEFINDER_MODE= (1 << 9),
-    FAILSAFE_MODE   = (1 << 10),
-    GPS_RESCUE_MODE = (1 << 11)
+    PASSTHRU_MODE   = (1 << 7),
```

Comment:
hey @ledvinap sorry for responding so late. im not sure as well tbh. the reason why i insert it at the end is because i wanted to see the flight mode in my DJI googles. using one of the the other unused modes did not work for me or i did not understand it.
currently you see "CHIR" in the googles when you flip the switch and when the chirp signal is finished you get a blinking warning saying "CHIRP IS FINISHED".
i think its important to have visual feedback about the chirp state, otherwise its really hard to tell if the chirp is still running or not, especially for higher frequencies. i will leave it for now, but certainly change it before i go from draft to ready.

Reply:
Ok, i swapped to the old autotune mode. Also removed shown flight mode in googles. User now needs warnings enabled to see if the chirp excitation is finished. But i think this if fine.